aaw/prob-auth/template.yaml

```diff
@@ -0,0 +1,23 @@
+apiVersion: templates.gatekeeper.sh/v1beta1
+kind: ConstraintTemplate
+metadata:
+  name: probauth
+spec:
+  crd:
+    spec:
+      names:
+        kind: ProBAuth
+  targets:
+    - target: admission.k8s.gatekeeper.sh
+      rego: |
+        package probauth
+        violation[{"msg": msg}] {
+          input.review.object.metadata.labels["data.statcan.gc.ca/classification"] == "protected-b"
```

Comment:
I think we can remove this from the check here and apply a selector to the constraint object. See https://github.com/StatCan/gatekeeper-policies/blob/feat-aaw-policies/aaw/metadata-restrictions/constraint.yaml#L29-L34 as an example of this.